### PR TITLE
Fix lint

### PR DESCRIPTION
**Pull request by jeremypw**
Needed before merging #537

### CODE REVIEW

File: plugins/pantheon-photos-publishing-extras/YandexPublishing.vala [hunk 1]
```diff
@@ -644,7 +644,7 @@ public class YandexPublisher : Spit.Publishing.Publisher, GLib.Object {
     private void start_web_auth () {
         host.set_service_locked (false);
 
-        web_auth_pane = new WebAuthPane (("http://oauth.yandex.ru/authorize?client_id=%s&response_type=token").printf (client_id));
+        web_auth_pane = new WebAuthPane (("http://oauth.yandex.ru/authorize?CLIENT_ID=%s&response_type=token").printf (CLIENT_ID));
```
Comment (ryonakano) [inline plugins/pantheon-photos-publishing-extras/YandexPublishing.vala]:
```suggestion
        web_auth_pane = new WebAuthPane (("http://oauth.yandex.ru/authorize?client_id=%s&response_type=token").printf (CLIENT_ID));
```

I didn't test, but I suppose this `CLIENT_ID` is a part of a query and does not need to (or should not) change?

File: src/Resources.vala [hunk 2]
```diff
@@ -379,51 +379,51 @@ private void fetch_lc_time_format () {
...
 }
 
 public string get_start_multimonth_span_format_string () {
-    if (START_MULTIMONTH_DATE_FORMAT_STRING == null) {
+    if (start_multimonth_date_format_string == null) {
         fetch_lc_time_format ();
     }
 
-    return START_MULTIMONTH_DATE_FORMAT_STRING;
+    return start_multimonth_date_format_string ;
```
Comment (ryonakano) [inline src/Resources.vala]:
```suggestion
    return start_multimonth_date_format_string;
```

File: plugins/pantheon-photos-publishing-extras/YandexPublishing.vala (hunk 1 above)
Comment (jeremypw) [reply plugins/pantheon-photos-publishing-extras/YandexPublishing.vala]: Yes, well spotted, this was autoreplaced - I'll fix.

File: src/Resources.vala (hunk 2 above)
Comment (jeremypw) [reply src/Resources.vala]: Done

**Pull request merged**